### ISSUE TITLE
crashes when called from cmd 

### ISSUE THREAD

**Issue by dkocich** [ClearTax/install-types#2]
node v12.13.0; npm v6.13.1

I installed your package globally and tried to call it in my folder - this is the result:
_
```
λ install-types
'typesync' is not recognized as an internal or external command,
operable program or batch file.
child_process.js:660
    throw err;
    ^

Error: Command failed: typesync
'typesync' is not recognized as an internal or external command,
operable program or batch file.

    at checkExecSyncError (child_process.js:621:11)
    at execSync (child_process.js:657:15)
    at init (C:\Users\username\AppData\Roaming\npm\node_modules\install-types\index.js:39:18)
    at Object.<anonymous> (C:\Users\username\AppData\Roaming\npm\node_modules\install-types\index.js:49:1)
    at Module._compile (internal/modules/cjs/loader.js:956:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
    at Module.load (internal/modules/cjs/loader.js:812:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1025:10)
    at internal/main/run_main_module.js:17:11 {
  status: 1,
  signal: null,
  output: [
    null,
    <Buffer >,
    <Buffer 27 74 79 70 65 73 79 6e 63 27 20 69 73 20 6e 6f 74 20 72 65 63 6f 67 6e 69 7a 65 64 20 61 73 20 61 6e 20 69 6e 74 65 72 6e 61 6c 20 6f 72 20 65 78 74 ... 49 more bytes>
  ],
  pid: 20212,
  stdout: <Buffer >,
  stderr: <Buffer 27 74 79 70 65 73 79 6e 63 27 20 69 73 20 6e 6f 74 20 72 65 63 6f 67 6e 69 7a 65 64 20 61 73 20 61 6e 20 69 6e 74 65 72 6e 61 6c 20 6f 72 20 65 78 74 ... 49 more bytes>
}
```

**Comment by maddhruv** [comment]
yeah! the issue is happening when I call it using `npx` also.
provided `install-types` or `typesync` is not installed globally

**Comment by dkocich** [comment]
I guess there should be mentioned https://github.com/ClearTax/install-types#note that we need to install it globally? This was not obvious to me at first until I started using `typesync` directly later... Closing

**Comment by maddhruv** [comment]
@dkocich Thanks for reporting, we have made a fix for this in `v1.0.1`
